source/docs/software/advanced-control/state-space/state-space-flywheel-walkthrough.rst

```diff
@@ -1,4 +1,123 @@
-State-Space Controller Examples
-===============================
+State-Space Controller Walkthrough
+==================================
 
+.. note:: Before following this tutorial, readers are recommended to have read an :ref:`docs/software/advanced-control/state-space/state-space-intro:Introduction to state-space control`.
 
+The goal of this tutorial is to provide "end-to-end" instruction on implementing a state-space controller on a flywheel.  By following this tutorial, readers will learn how to:
```

Comment:
Replace with "to provide an 'end-to-end' set of instructions on..." or "to provide an 'end-to-end' tutorial on..."